mlflow/R/mlflow/R/model.R

```diff
@@ -146,23 +149,22 @@ mlflow_load_model <- function(model_path, flavor = NULL, run_id = NULL) {
 #' jsonlite::write_json(iris, "iris.json")
 #'
 #' # predict existing model from json data
-#' mlflow_rfunc_predict("mlflow_roundtrip", "iris.json")
+#' mlflow_rfunc_predict("file:mlflow_roundtrip", "iris.json")
```

Comment:
`file:/`

Reply:
Talked offline, "file:" is the correct prefix for relative paths.